### PR TITLE
Added rooms to castle AB#183

### PR DESCRIPTION
**Pull request by kaashis**

### CODE REVIEW

File: src/com/zombiecastlerush/building/Castle.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package com.zombiecastlerush.building;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Castle {
+    //create a list of rooms in castle
+    List<Room>  castleRooms = new ArrayList<Room>();
+
```
Comment (wang-xiaotian) [inline src/com/zombiecastlerush/building/Castle.java:10]: how does the list provide an access point to this Castle? Are you going to use Map(our Map class not data structure) to provide access point？

**Review by wang-xiaotian — APPROVED** [review]

File: src/com/zombiecastlerush/building/Castle.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package com.zombiecastlerush.building;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class Castle {
+    //create a map of rooms in castle
+    private Map<Room, List<Room>> castleRooms = new HashMap<Room,List<Room>>();
```
Comment (chaynsaw) [inline src/com/zombiecastlerush/building/Castle.java]: Minor quibble, fix spacing. I suggest maybe `new HashMap<Room, List<Room>>();` for consistency sake. Non-blocking.

**Review by chaynsaw — APPROVED** [review]

File: src/com/zombiecastlerush/building/Castle.java (hunk 1 above)
Comment (chaynsaw) [reply src/com/zombiecastlerush/building/Castle.java:10]: Aashis: Castle has a Map

**Comment by chaynsaw** [comment]
Push latest changes and merge. :)

**Pull request merged**